CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # CHANGELOG
+## 3.4.0
+#### New Functionality
+
+- Can now talk to MarkLogic using OAuth token (MarkLogic Server version 11.2.0 or higher needed).
+- Can now submit optic update plans to /v1/rows/update (MarkLogic Server version 11.2.0 or higher needed).
+- Can now use PatchBuilder for patching documents (MarkLogic Server version 11.2.0 or higher needed).
+- Can now use onError (MarkLogic Server version 11.2.0 or higher needed).
```

Comment:
Same thing here, need to give the user context: "Can now capture errors from executing an Optic plan by including the new `onError` operator."